UE/Application/Ports/UserPort.cpp

```diff
@@ -53,7 +54,9 @@ void UserPort::showConnected()
             showSmsList();
         } else if(menu.getCurrentItemIndex().second == 2){
             setupCallReceiver();
-        };
+        } else if (menu.getCurrentItemIndex().second == 0) {
+            showSmsCompose();
+        }
```

Comment:
You should add this to the beginning of the `if` instruction to maintain the order.

Reply:
I'll change that. But imo it should be refactored to some switch instruction if we will have some time in the future.